src/Particle/ParticleBase/tests/test_random_seq.cpp

```diff
@@ -102,4 +104,42 @@ TEST_CASE("gaussian random input zero", "[particle_base]")
   REQUIRE(a[0] == Approx(0.0));
   REQUIRE(a[1] == Approx(0.0));
 }
+
+TEST_CASE("makeGaussRandomWithEngine(MCCoords...)", "[particle_base]")
+{
+  int size_test = 7;
+  std::vector<double> gauss_random_vals(size_test * 3 + (size_test * 3) % 2 + size_test );
+  {
+    StdRandom<double> rng;
+    makeGaussRandomWithEngine(gauss_random_vals, rng);
+  }
+
+  auto checkRs = [&](auto& rs) {
+    for (int i = 0; i < size_test; ++i)
+      CHECK(TinyVector<double, 3>(gauss_random_vals[3 * i], gauss_random_vals[3 * i + 1],
+                                  gauss_random_vals[3 * i + 2]) == rs[i]);
+  };
+
+  MCCoords<CoordsType::POS> mc_coords_rs;
```

Comment:
Get the deltas for MCCoords in this fashion.  generateDeltas ended up just being a wrapper for makeGaussRandomWithEngine.

Reply:
sounds good. was just making sure the intention wasn't to still provide that wrapper